### ISSUE TITLE
Rewrite parsers

### ISSUE THREAD

**Issue by bmcfee** [marl/jams-data#1]
[repost from https://github.com/marl/jams/issues/31 ]

It is time to rewrite the dataset parsers, so that they work again with the current JAMS version (`v0.2.2`?).
- [x] ADC2004Melody
- [ ] Billboard
- [x] Isophonics
- [x] JKU Patterns
- [ ] MedleyDB
- [x] MIREX05Melody
- [ ] Rockcorpus
- [x] SALAMI
- [x] SMC
- [ ] TMC323
- [x] CAL500
- [x] CAL10K
- [ ] MSD
- [ ] IRMAS
- [ ] RWC POP (smf midi --> melody only `pitch_midi`)
- [ ] MIDI2JAMS

@bmcfee already (re?)wrote the [SMC one](https://github.com/marl/jams/blob/develop/parsers/smc_parser.py), and I'm working on SALAMI and Isophonics.

I'm also gonna assign this to some of you, because my roots come from Spain, and we are used to [dictatorship](http://en.wikipedia.org/wiki/Francisco_Franco)[(s)](http://en.wikipedia.org/wiki/Philip_V_of_Spain) there.

**Comment by justinsalamon** [comment]
I'll hopefully get to the adc2004 and mirex05 melody datasets some time soon. Also, do we have any parser that converts MIDI to JAMS? I need this for a project, so if there's something doing something similar let me know. If not I guess I'll have to give it a crack.

**Comment by bmcfee** [comment]
> Also, do we have any parser that converts MIDI to JAMS?

We don't, but it shouldn't be difficult to hack using pretty_midi.

**Comment by justinsalamon** [comment]
yeah that's what I thought. OK, I'll give it a crack (for a first iteration I just need to extract the melody track from the MIDI file and covert into JAMs, but I'll see if it's easy to kill the two birds with one stone).

**Comment by justinsalamon** [comment]
I've added two converters to the todo list: "RWC POP (smf midi --> melody only `pitch_midi`)" and "MIDI2JAMS". The former is pretty specific, and will extract the melody track (only) from the RWC POP midi files and save it to a JAMS `pitch_midi` annotation. The latter will be a generic "pour all the notes in the MIDI file into a JAMS file".
